app/src/main/java/dev/shreyaspatil/covid19notify/ui/main/MainActivity.kt

```diff
@@ -66,6 +68,29 @@ class MainActivity : AppCompatActivity() {
         mStateAdapter.clickListener = this::navigateToStateDistrictScreen
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.toolbar_menu, menu)
+        //Setting an Icon tint so that it can be backward compatible
+        val item = menu?.findItem(R.id.menuSettings)
+        val drawableWrap = DrawableCompat.wrap(item?.icon!!).mutate()
+        DrawableCompat.setTint(drawableWrap, ContextCompat.getColor(this, R.color.colorAccent))
+        item.icon = drawableWrap
+        return true
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        return when (item.itemId) {
+            R.id.menuSettings -> {
+                Intent(this, SettingsActivity::class.java).also {
+                    startActivity(it)
```

Comment:
Since there is only one option in settings i.e. Change theme. So it'll good to have a direct switching operation from this menu instead of creating separate activity or fragment for a single task. See [this](https://github.com/PatilShreyas/Foodium/blob/master/app/src/main/java/dev/shreyaspatil/foodium/ui/main/MainActivity.kt) for reference.

Reply:
I ll have links and about section thats why it is good to have it ,I asked for your help way before ...but u didn't responded so I did it what was good for both of us.

Reply:
Oh sorry for that!